### PR TITLE
whatis: add Japanese translation, update page

### PR DESCRIPTION
**Pull request by yuiseki**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** Unknown

I would prefer to work on commands that are useful for beginners first.

In this pull request, I've translated `pages/linux/whatis.md` file to `pages.ja/linux/whatis.md` file as Japanese.

In addition of the above, I've added the example of `--locale` option of `whatis` command.
**Question**: If this is the case, should I make changes to the original English version as well?

I believe that `whatis` is just as useful for beginners as this `tldr`, so I really wanted to translate it into Japanese.

### CODE REVIEW

**Comment by kbdharun** [comment]
> Question: If this is the case, should I make changes to the original English version as well?

Yes, it is generally suggested to apply any new additions to English version too.

File: pages.ja/linux/whatis.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# whatis
+
+> マニュアルページから、一行の説明文を表示します。
+> 詳しくはこちら: <https://manned.org/whatis>.
+
+- manページの説明文を表示する:
+
+`whatis {{command}}`
```
Comment (kbdharun) [inline pages.ja/linux/whatis.md:8]:
```suggestion
`whatis {{コマンド}}`
```

File: pages.ja/linux/whatis.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- 説明文を行末で切らないで表示する:
+
+`whatis --long {{command}}`
```
Comment (kbdharun) [inline pages.ja/linux/whatis.md:12]:
```suggestion
`whatis --long {{コマンド}}`
```

File: pages.ja/linux/whatis.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`whatis --wildcard {{net*}}`
+
+- 正規表現でmanページの説明文を検索する:
+
+`whatis --regex '{{wish[0-9]\.[0-9]}}'`
+
+- 言語を指定して説明文で表示する (`manpage-{{ja}}` パッケージが必要です)
+
+`whatis --locale={{ja}} {{command}}`
```
Comment (kbdharun) [inline pages.ja/linux/whatis.md:24]:
```suggestion
`whatis --locale={{ja}} {{コマンド}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages.ja/linux/whatis.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- globパターンにマッチするすべてのコマンドの説明文を表示する:
+
+`whatis --wildcard {{net*}}`
+
+- 正規表現でmanページの説明文を検索する:
+
+`whatis --regex '{{wish[0-9]\.[0-9]}}'`
+
+- 言語を指定して説明文で表示する (`manpage-{{ja}}` パッケージが必要です)
```
Comment (kbdharun) [inline pages.ja/linux/whatis.md:22]:
```suggestion
- 言語を指定して説明文で表示する (`manpage-{{ja}}` パッケージが必要です):
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**